Serato-Now-Playing/serato_now_playing.py

```diff
@@ -513,28 +435,30 @@ def on_filebutton_clicked(self):
             startdir = os.path.dirname(startfile)
         else:
             startdir = '.'
-        filename = QFileDialog.getOpenFileName(self.window, 'Open file',
+        filename = QFileDialog.getSaveFileName(self.window, 'Open file',
                                                startdir, '*.txt')
         if filename:
             self.fileEdit.setText(filename[0])
 
     def on_txttemplatebutton_clicked(self):
         ''' file button clicked action '''
+        global BUNDLE_DIR
+
         startfile = self.txttemplateEdit.text()
         if startfile:
             startdir = os.path.dirname(startfile)
         else:
-            startdir = '.'
+            startdir = os.path.join(BUNDLE_DIR, "templates")
```

Comment:
Should there be a 'reset defaults' button now?  How does a user get back to this if they want one of the built-ins?